docs/source/notes/distributed_autograd.rst

```diff
@@ -113,7 +113,12 @@ From the user's perspective the autograd context is setup as follows:
   import torch.distributed.autograd as dist_autograd
   with dist_autograd.context() as context_id:
     loss = model.forward()
-    dist_autograd.backward(loss)
+    dist_autograd.backward(context_id, loss)
```

Comment:
is it mandatory to specify the `context_id` here, or will it infer the id if not provided?

Reply:
Yep, the context_id is required now: https://github.com/pytorch/pytorch/blob/master/torch/csrc/distributed/autograd/init.cpp